src/Build.UnitTests/ChangeWaves_Tests.cs

```diff
@@ -19,200 +19,146 @@ public ChangeWaves_Tests(ITestOutputHelper output)
             _output = output;
         }
 
-        [Theory]
-        [InlineData("16.8")]
-        [InlineData("16.10")]
-        [InlineData("17.0")]
-        [InlineData("25.87")]
-        [InlineData("102.87")]
-        public void EnableAllFeaturesBehindChangeWavesEnablesAllFeaturesBehindChangeWaves(string featureWave)
+        /// <summary>
+        /// Helper function to build a simple project based on a particular change wave being set.
+        /// Call SetChangeWave on your TestEnvironment before calling this function.
+        /// </summary>
+        private void buildSimpleProjectAndValidateChangeWave(TestEnvironment testEnvironment, Version waveToCheck, Version changeWaveShouldUltimatelyResolveTo, params string[] warningCodesLogShouldContain)
```

Comment:
can you do doc comments for the arguments too? Not sure I totally grok.

Reply:
Part of the confusion may be that this helper function basically does all the checking for you. I added the comments.